apps/docs/pages/guides/database/api.mdx

```diff
@@ -0,0 +1,418 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'api',
+  title: 'Serverless APIs',
+  description: 'Auto-generating and Realtime APIs.',
+  sidebar_label: 'Overview',
+  video: 'https://www.youtube.com/v/rPAJJFdtPw0',
+}
+
+Supabase auto-generates three types of API directly from your database schema.
+
+- REST - interact with your database through a restful interface.
+- Realtime - listen to database changes.
+- GraphQL - manipulate your database using a graph-like query language
+
+The APIs are:
+
+- **Instant and auto-generated.** <br />As you update your database the changes are immediately accessible through your API.
+- **Self documenting.** <br />Supabase generates documentation in the Dashboard which updates as you make database changes.
+- **Secure.** <br />The API is configured to work with PostgreSQL's Row Level Security, provisioned behind an API gateway with key-auth enabled.
+- **Fast.** <br />Our benchmarks for basic reads are more than 300% faster than Firebase. The API is a very thin layer on top of Postgres, which does most of the heavy lifting.
+- **Scalable.** <br />The API can serve thousands of simultaneous requests, and works well for Serverless workloads.
+
+## REST API [#rest-api-overview]
+
+Supabase provides a RESTful API using [PostgREST](https://postgrest.org/). This is a very thin API layer on top of Postgres.
+It provides everything you need from a CRUD API at the URL `https://<project_ref>.supabase.co/rest/v1/`.
+
+Features:
```

Comment:
The REST interface is automatically reflected from your database's schema and supports: